src/lib/containers/FullWidthAlert.tsx

```diff
@@ -27,7 +27,7 @@ export type AlertButtonConfig = {
 export interface FullWidthAlertProps extends AlertProps {
   variant: string
   title?: string
-  description?: string
+  description?: React.ReactNode
```

Comment:
ImposeRestrictionModal (called when a user clicks "Add Conditions for Use") passes a custom `Typography` block instead of a string here. Not sure why the build didn't break until this change.